### ISSUE TITLE
Doesn't work with Google Groups

### ISSUE THREAD

**Issue by djbclark** [danny0838/firefox-scrapbook-autosave#2]
This doesn't auto-save Google Groups messages.

Only thing I can think is maybe it doesn't like "#!" in the URL.

URLs are like:
https://groups.google.com/forum/m/#!topic/GROUPNAME/RANDOMSTRING

**Comment by danny0838** [comment]
You are right. URLs with difference only after "#" are actually "same" page while using different in-page anchor. It's a  common usage for dynamic web pages triggered by AJAX, and ScrapBook X AutoSave cannot detect it since there is no real page switching, and manually capture would be required.
